root/components/Aliases/AliasTableRow.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const Frag = require('../Frag');
+const isolateText = require('../../static/scripts/common/utility/isolateText');
+const formatDate = require('../../static/scripts/common/utility/formatDate');
+const entityHref = require('../../static/scripts/common/utility/entityHref');
+const locales = require('../../static/scripts/common/constants/locales');
+const {l} = require('../../static/scripts/common/i18n');
+
+type Props = {
+  +alias: AliasT,
+  +allowEditing: boolean,
+  +entity: $Subtype<CoreEntityT>,
+  +row: string,
+};
+
+const AliasTableRow = ({alias, allowEditing, entity, row}: Props) => (
+  <tr className={row} key={alias.id}>
+    <td colSpan={alias.name === alias.sort_name ? 2 : 1}>
+      {alias.editsPending ? <span className="mp">{isolateText(alias.name)}</span> : isolateText(alias.name)}
+    </td>
+    {alias.name === alias.sort_name
+      ? null
+      : <td>{isolateText(alias.sort_name)}</td>}
+    <td>{formatDate(alias.begin_date)}</td>
+    <td>{alias.ended ? alias.end_date ? formatDate(alias.end_date) : l('Unknown') : null}</td>
+    <td>{alias.typeName}</td>
+    <td>
+      {alias.locale ? locales[alias.locale] : null}
+      {alias.primary_for_locale ? l(' ({primary})', {__react: true, primary: <span className="comment">{l('primary')}</span>}) : null }
```

Comment:
I still think it should use `bracketed`, but otherwise can we change the string to `{text}` so they share the same translation?

Reply:
Ah yes, forgot about this. Let me fix it.

Reply:
Done :)